menu_vypocty.py

```diff
@@ -0,0 +1,34 @@
+def menu():
+    print("\nZvolte jednu z možností:")
+    print("Q - Quit (Ukončit program)")
+    print("S - Square (Umocnit číslo na druhou)")
+    print("D - Double (Vynásobit číslo dvěma)")
+    print("H - Half (Vydělit číslo dvěma)")
+    print("P - Print (Zobrazit hodnotu čísla n)")
+
+# Hlavní část programu
+n = float(input("Zadejte číslo: "))

```

Comment:
If the user enters non-numeric input, the program crashes.

Reply:
Good point, we should handle this scenario by some nicer output. If the app is in Czech lang, output should be in Czech as well.

Reply:
Also, `n` is usually used for integers, `f`, or better `user_input_number` can be clearer